Source/Scene/Cesium3DTileset.js

```diff
@@ -984,51 +988,60 @@ define([
                 } else {
                     // Tile does not meet SSE.
 
+                    // Get visibility for all children. Check if any visible children are not loaded.
+                    var allVisibleChildrenLoaded = true;
+                    var someVisibleChildrenLoaded = false;
+                    for (k = 0; k < childrenLength; ++k) {
+                        child = children[k];
+                        child.parentPlaneMask = planeMask;
+                        child.planeMask = child.visibility(frameState.cullingVolume);
+                        // If the child is visible...
```

Comment:
Instead of having this comment everywhere consider adding a static `isVisible` function and pass it `child.planeMask` so everything is super obvious and explicit.